### ISSUE TITLE
Can't access the solvers

### ISSUE THREAD

**Issue by Certhas** [conda-forge/sundials-feedstock#7]
I tried installing this package but unfortunately I can't seem to get it (or the assimulo package for that matter) to work. When I try to load the library I get the following error:

```
In [1]: import ctypes

In [2]: testlib = ctypes.CDLL("libsundials_ida.so")
---------------------------------------------------------------------------
OSError                                   Traceback (most recent call last)
<ipython-input-2-6338dd4a6475> in <module>()
----> 1 testlib = ctypes.CDLL("libsundials_ida.so")

/home/frank/anaconda2/lib/python2.7/ctypes/__init__.pyc in __init__(self, name, mode, handle, use_errno, use_last_error)
    360 
    361         if handle is None:
--> 362             self._handle = _dlopen(self._name, mode)
    363         else:
    364             self._handle = handle

OSError: /home/frank/anaconda2/lib/python2.7/lib-dynload/../../libsundials_ida.so: undefined symbol: dscal_

```

Potentially related, assimulo doesn't work:
https://github.com/conda-forge/assimulo-feedstock/issues/4

**Comment by bjodah** [comment]
Not sure what's going on there, but `dscal` is a BLAS function. I haven't used assimulo myself but I've been using conda-forge/sundials under linux successfully, see e.g. this CI log: https://travis-ci.org/chemreac/chemreac/builds/199563004

**Comment by jschueller** [comment]
they do not link lapack: b59eabc

**Comment by jschueller** [comment]
should be fixed by #8
